inc/tpl/firewall-clearcache.html.tpl

```diff
@@ -20,6 +20,30 @@ jQuery(document).ready(function ($) {
             $('#firewall-clear-cache-response').html(data);
         });
     });
+    
+    $('#firewall-clear-cache-path-button').on('click', function (event) {
+        event.preventDefault();
+
+        var button = $(this),
```

Comment:
is that... a `var` usage?  see comment where i ask what year this is.  i think i woke up in the early 2000s.